scripts/zones/Bostaunieux_Oubliette/npcs/qm_ethereal_droplet.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Area: Reisenjima (291)
+-- NPC: ???
+-- Notes: Grants "Ethereal droplet" temporary item.
+-----------------------------------
+local ID = require("scripts/zones/Reisenjima/IDs")
+require("scripts/globals/items")
+require("scripts/globals/status")
+-----------------------------------
+
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    if player:hasItem(xi.items.ETHEREAL_DROPLET, xi.inv.TEMPITEMS) then
+        player:messageSpecial(ID.text.NOTHING_OUT_OF_ORDINARY)
+    else
+        player:addTempItem(xi.items.ETHEREAL_DROPLET, 1)
+        player:messageSpecial(ID.text.ITEM_OBTAINED, xi.items.ETHEREAL_DROPLET)
```

Comment:
Not such a big deal, but we should use
```suggestion
        player:addTempItem(xi.items.ETHEREAL_DROPLET, 1)
        player:messageSpecial(ID.text.ITEM_OBTAINED, xi.items.ETHEREAL_DROPLET)
```
And add the nothing out of the ordinary to defaultActions.
Although this is way far out of era, and shouldn't matter for ASB